UnityGLTF/Assets/UnityGLTF/Runtime/Plugins/GLTFSerialization/Schema/GLTFProperty.cs

```diff
@@ -21,7 +21,9 @@ public class GLTFProperty
 			{ KHR_materials_iridescence_Factory.EXTENSION_NAME, new KHR_materials_iridescence_Factory() },
 			{ KHR_materials_specular_Factory.EXTENSION_NAME, new KHR_materials_specular_Factory() },
 			{ KHR_materials_clearcoat_Factory.EXTENSION_NAME, new KHR_materials_clearcoat_Factory() },
-      		{ MSFT_LODExtensionFactory.EXTENSION_NAME, new MSFT_LODExtensionFactory() }
+      		{ MSFT_LODExtensionFactory.EXTENSION_NAME, new MSFT_LODExtensionFactory() },
+            { KHR_draco_mesh_compression_Factory.EXTENSION_NAME, new KHR_draco_mesh_compression_Factory() },
```

Comment:
Please fix the indenting

Reply:
825268c674f3de83335c782103f09cc3f77678f1